lightning/src/chain/channelmonitor.rs

```diff
@@ -155,6 +155,17 @@ pub enum MonitorEvent {
 	/// A monitor event containing an HTLCUpdate.
 	HTLCEvent(HTLCUpdate),
 
+	/// Indicates we broadcasted the channel's latest commitment transaction and thus closed the
+	/// channel. Holds information about the channel and why it was closed.
+	HolderForceClosedWithInfo {
```

Comment:
Should we name this `HolderForceClosed` and rename the existing variant to `HolderForceClosedWithoutInfo` instead? I'm somewhat indifferent on doing this now but seems like the one used going forward shouldn't have a suffix.

Reply:
imo it makes more sense like this and it can be renamed when `HolderForceClosed` is removed